src/views/admin-users-listing.phtml

```diff
@@ -65,17 +65,17 @@ require_once('partials/header.phtml');
 
     <!--Pagination block-->
     <nav aria-label="Page navigation example">
-        <?php if ($count > 5):
+        <?php if ($paginator->getTotalResults() > 5):
```

Comment:
try to accomodate the idea that you might, in a future issue, allow the user to set the number of results/page, thus that 5 should not be harcoded